README.md

```diff
@@ -80,6 +98,13 @@ npm test
 - LinkedIn: [@HenryOdongo](https://www.linkedin.com/in/henry-odongo-91b830182/)
 - Website: [@Portfolio](https://odongo006.github.io/My-Portfolio-Project/)
 
+👤 **Mert**
+
+- GitHub: [@mertkantaroglu](https://github.com/mertkantaroglu)
+- Twitter: [@]()
+- LinkedIn: [@]()
+- Website: [@]()
```

Comment:
- [ ] Please kindly add links to your social media accounts. This makes your README look professional and also you can easily be contacted.